source/sync/migrate-sync-modes.txt

```diff
@@ -0,0 +1,362 @@
+doesn't require any changes to your client app code.
+
+Migration results in Partition-Based Sync clients communicating with a
+Flexible Sync backend. We recommend eventually :ref:`migrating your client app 
+<realm-sync-migrate-client>` to also use Flexible Sync instead of
+Partition-Based Sync.
+
+Requirements
+------------
+
+Currently, the Kotlin and Swift SDKs can migrate from Partition-Based Sync to
+Flexible Sync. The Java, .NET, Node.js, and React Native SDKs cannot perform
+the migration.
+
+App Services App:
+
+- Currently use Partition-Based Sync
+
+Linked Atlas Cluster:
+
+- Use MongoDB 5.0 or later for Flexible Sync compatibility
+- Enable :atlas:`storage auto-scaling </cluster-autoscaling/#configure-auto-scaling-options>`
+- Be an M10+ Cluster Tier
+- Have time-based :manual:`oplog </core/replica-set-oplog/>` setting enabled
+
+Client apps:
+
+- Minimum SDK version:
+
+  - Realm Kotlin SDK v1.8.0
+  - Realm Swift SDK v10.39.0
+- Have a :ref:`Client Reset Handler <client-resets>` configured.
+- Remove any code in a client app that relies on progress notifications because Flexible Sync does not support them
+
+.. _realm-sync-migrate-what-to-know:
+
+What to Know Before Migrating
+-----------------------------
+
+Migrating your App's Sync mode involves changes that affect both client apps and
+your App's backend. You should be aware of the effects and plan ahead.
+
+- Client apps that don't meet the minimum SDK version requirement will not be
+  able to sync after migration.
+- How long a migration takes is directly proportional to the amount of data that
+  needs to be migrated. The more data, the longer it will take to migrate.
+- Migrations can be :ref:`canceled <realm-sync-migrate-cancel>` during the 
+  migration or :ref:`reverted <realm-sync-migrate-revert>` within a certain 
+  timeframe after the migration is finished.
+- Progress notifications will not work after migration because Flexible Sync does
+  not support them.  
+- Client apps that were connected to the Partition-Based Sync backend will 
+  experience a client reset after the backend is migrated to Flexible Sync. You 
+  should use a :ref:`client reset <handle-a-client-reset>` handler with recovery enabled.
+- Creating new tables in a migrated Flexible Sync backend using 
+  :ref:`Development Mode <development-mode>` from a Partition-Based Sync client
+  may lead to unexpected behavior.
+
+.. _realm-sync-migrate-stages:
+
+Migration Stages
+~~~~~~~~~~~~~~~~
+
+You can see your migration's stage in the App Services UI. There are three stages
+to a Sync mode migration:
+
+#. **Syncing Partition-Based Sync metadata** 
+   
+   At this stage, client apps can connect to your App's backend, but local writes 
+   will not sync. These writes will be lost if you don't have a 
+   :ref:`client reset handler <handle-a-client-reset>` to recover unsynced changes.
+	 
+   Clients can also receive updates to synced collections, but they won't
```

Comment:
I'd say "clients will receive updates to synced collections, but won't be able to send uploads or receive upload acknowledgements".